en/codelists-guides/codelist-management.rst

```diff
@@ -2,108 +2,115 @@ IATI Codelist Management
+**Modification Schedule: Non-Embedded Codelists - Replicated**
 
-In most circumstances the decision to alter these codelists is straightforward. In rare cases, the team may decide to seek further clarification, or to consult further on whether or not to adopt a change. If so, a timetable outlining the process and time scales will be issued.
+Codes in Replicated Codelists may be:
 
-Once a change has been agreed, a GitHub issue will be created and will be linked to from the forum post.
+- Added - at any point in time, subject to applicable notification by the third-party
+- Modified - at any point in time, subject to applicable notification by the third-party
+- Withdrawn - at any point in time, subject to applicable notification by the third-party
+- Removed - never
 
-When the change has been made and accepted into the master branch of the `Non-embedded Codelist repository <https://github.com/IATI/IATI-Codelists-NonEmbedded>`__, the change will be 'live' to anyone who consumes these codelists directly from Github or via the API. 
+Examples of Replicated Codelists include: DAC 5-Digit, Organisation Registration Agency
 
-At this point the technical team will:
+Non-Embedded Codelists - External
+----------------------
 
- * Update the `Non-Embedded Codelist changelog record <http://iatistandard.org/upgrades/nonembedded-codelist-changelog>`__
- * Regenerate the IATI Standard Website.
- * Mark the forum post as 'Done'
+External Codelists are a collection of codes managed by a third-party organisation. The IATI Technical Team does not maintain a correspoding copy in a document that conforms to the Codelist Schema.
 
-A :doc:`summary of all changes to all non-emebedded </upgrades/nonembedded-codelist-changelog/>` codelists is published on the IATI Standard website. They can also be found via the list of commits to the GitHub repository.
+**Modification Schedule: Non-Embedded Codelists - External**
 
-Special Cases
-*************
+Codes in External Codelists may be:
 
-File Format
-  The IATI Technical Team will use its judgement to assess the importance of a change to the file format list and whether or not an announcement is required (i.e. the list may alter to stay inline with the external source, but announcements will not necessarily be made).
+- Added - at any point in time
+- Modified - at any point in time
+- Withdrawn - at any point in time
+- Removed - at any point in time
```

Comment:
I suppose the maintainers of external codelists may have their own upgrade processes.

Reply:
I presume so. And they are outside of our remit.

Reply:
So I mean: rather than saying "at any point in time" this could say "according to the owner’s schedule" or something like that.